server/brunch-config.js

```diff
@@ -0,0 +1,69 @@
+exports.config = {
```

Comment:
way do we need brunch? The server is already configed with webpack for assets bundling.